woocommerce/woocommerce-android
# WooPayments setup: tracks

Closes: #9717 

### Description
This PR adds tracking for the WooPayments setup:

| Event name | Trigger | Properties |
| --- | --- | --- |
| store_onboarding_task_tapped (**Already implemented**) | When the user taps on a task in the store onboarding card. | – task:woocommerce-payments |
| store_onboarding_task_completed | When the user completes a task in the store onboarding card in the app. | – task: woocommerce-payments |
| store_onboarding_wcpay_begin_setup_tapped | When the user taps on the button “Begin setup” on the flow start screen. | |  
| store_onboarding_wcpay_terms_continue_tapped | When the user taps on “Continue” on the wcpay terms screen. |

### Testing instructions
1. Prepare a fresh store that satisfies:
     - US as country.
     - WooPayments installed.
     - WCPay dev tools installed.
2. Open the app the sign in.
3. Start the WooPayments task.
4. Confirm the following event: `Tracked: store_onboarding_task_tapped, Properties: {"task":"woocommerce-payments"...}`
5. Tap on Begin setup, and confirm the event: `Tracked: store_onboarding_wcpay_begin_setup_tapped`
6. Tap on Continue and confirm the event: `Tracked: store_onboarding_wcpay_terms_continue_tapped`
7. Finish the setup on the WebView (Check P91TBi-4BH-p2 for more details on using test data).
8. When the setup is done, confirm the following event: `Tracked: store_onboarding_task_completed, Properties: {"task":"woocommerce-payments"...}`
